src/udev/udevadm-trigger.c

```diff
@@ -526,6 +526,8 @@ int trigger_main(int argc, char *argv[], void *userdata) {
                 if (r < 0)
                         return log_error_errno(r, "Failed to create device monitor object: %m");
 
+                (void) sd_device_monitor_set_receive_buffer_size(m, 128*1024*1024);
```

Comment:
we actually set this in most of our code now. Should we maybe just set this internally in sd-device, so that the problem goes away?

i mean, socket memory is these days accounted to the process that owns a socket, hence we shouldn't be too concerned that this might waste memory.